src/client/components/Headline/Headline.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import styled, { StyledComponent } from '@emotion/styled'
+import { fonts, colorsV3 } from '@hedviginsurance/brand'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+
+const { FAVORIT } = fonts
+const { gray900 } = colorsV3
+
+export type Props = {
```

Comment:
nit:
```suggestion
export type HeadlineProps = {
```

Otherwise we'd need to rename it in the scenario we import this type inside a file that already define a `Props` type (which seems to be the case on most of our components files).

Reply:
I think we generally name the main props object type in a file `Props` - but if it feels like it's very annoying to import it `as HeadlineProps` we can of course adopt a pattern where we always name the props object type in a more specific way..

Reply:
No, It's not very annoying do the renaming 😄 
Let's keep it as it its